seed/challenges/01-responsive-web-design/basic-css.json

```diff
@@ -1049,8 +1049,8 @@
         "For example, if we wanted to create a red, 5 pixel border around an HTML element, we could use this class:",
         "<blockquote>&#60;style&#62;<br>&nbsp;&nbsp;.thin-red-border {<br>&nbsp;&nbsp;&nbsp;&nbsp;border-color: red;<br>&nbsp;&nbsp;&nbsp;&nbsp;border-width: 5px;<br>&nbsp;&nbsp;&nbsp;&nbsp;border-style: solid;<br>&nbsp;&nbsp;}<br>&#60;/style&#62;</blockquote>",
         "<hr>",
-        "Create a class called <code>thick-green-border</code> that puts a 10-pixel-wide green border with a style of <code>solid</code> around an HTML element, and apply that class to your cat photo.",
-        "Remember that you can apply multiple classes to an element by separating each class with a space within its <code>class</code> attribute. For example:",
+        "Create a class called <code>thick-green-border</code>. This class should add a 10px, solid, green border around a HTML element. Apply the class to your cat photo.",
```

Comment:
The correct indefinite article in American English to proceed HTML is "an", as was used in the original copy.